src/components/discover-sheet/ListsSection.js

```diff
@@ -46,26 +46,27 @@ const fetchTrendingAddresses = async coingeckoIds => {
 
 const ListButton = styled(ButtonPressAnimation).attrs({
   scaleTo: 0.96,
-})`
-  margin-right: 16px;
-  ${({ selected, theme: { colors } }) =>
-    selected
-      ? `
-        background-color: ${colors.alpha(colors.blueGreyDark, 0.06)};
-        border-radius: 12px;
-        height: 30px;
-        padding-horizontal: 8px;
-        padding-top: ${ios ? 6.5 : 4.5}px;
-      `
-      : `
-        padding-top: ${ios ? 6.5 : 4.5}px;
-      `}
-`;
+})(({ selected, theme: { colors } }) =>
+  selected
+    ? {
+        backgroundColor: colors.alpha(colors.blueGreyDark, 0.06),
+        borderRadius: 12,
+        height: 30,
+        marginRight: 16,
+        paddingHorizontal: 8,
+        paddingTop: ios ? 6.5 : 4.5,
+      }
+    : {
+        marginRight: 16,
```

Comment:
Looks like this was accidentally moved into the ternary and needs to be hoisted out.